### ISSUE TITLE
add_risk_table: min at risk = 0

### ISSUE THREAD

**Issue by SHAESEN2** [openpharma/visR#91]
## Issue description
When the argument `min_at_risk` = 0, I am expecting the risk table to extend the entire x axis

## Solution
I played around a bit and found an easy solution:
```{r}
  if (min_at_risk > 0) {
  max_time <-
    tidy_object %>%
    dplyr::filter(n.risk >= min_at_risk) %>%
    dplyr::group_by(strata) %>%
    dplyr::summarize(max_time = max(time)) %>%
    dplyr::ungroup() %>%
    dplyr::summarize(min_time = min(max_time)) %>%
    dplyr::pull(min_time)
  } else {
    max_time = max(time_ticks)
  }
```

**Comment by SHAESEN2** [comment]
The above solution needs adjustment as we need to take the max time per strata into account.

I see two options here:
 - Since we don't have information beyond what is available we should not display information along the entire x-axis
    eg max time to = 198 hours so we leave information at 200 empty.
- LOCF from time point 198 hours towards last time tick of 200 hours. If we want to do this, we need to decide on the imputation method: do we use LOCF or follow the strong assumption of summary.survfit that puts everything to 0

**Comment by SHAESEN2** [comment]
Fixed with PR #147 by looking at one tick beyond when the min risk set falls between breaks
